src/service/event-manager.service.ts

```diff
@@ -17,7 +17,10 @@
  limitations under the License.
  */
 import { Injectable } from '@angular/core';
-import { Observable, Observer, Subscription} from 'rxjs/Rx';
+import { Observable } from 'rxjs/Observable';
+import { Observer } from 'rxjs/Observer';
+import { Subscription } from 'rxjs/Subscription';
+import 'rxjs/add/operator/filter';
```

Comment:
Why is this needed?

Reply:
filter is used on Line 53 https://github.com/ruddell/ng-jhipster/blob/8c56718f84fa49c1b72f0d71e35888c419230da4/src/service/event-manager.service.ts#L53

Reply:
The full explanation is in the referenced issue.  By only importing the operators that are used, the import size of rxjs is massively reduced.

Reply:
Ok thanks